Santouka/css/main.scss

```diff
@@ -0,0 +1,67 @@
+@import url(https://fonts.googleapis.com/css?family=Unica+One);
+
+body {
+  font: 100%/1.4 Unica One;
+  text-transform: uppercase;
+  background: #222;
+  color: whitesmoke;
+  height: 100vh;
+  overflow: hidden;
+}
+
+h1 {
+  font-size: 40px;
+  text-align: center;
+  transition: opacity .4s;
+}
+
+.layer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  color: black;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  transition: transform .4s;
+  transform: 
+      perspective(800px) 
+      scale(1) 
+      translateX(0)
+      rotateY(0);
+  transform-style: preserve-3d;
+
+}
+
+
```

Comment:
same.
generally, class based approach is better than tag approach.

Reply:
I fixed it !